### PR TITLE
Add batch_norm option to DNNLinearCombinedEstimator

### PR DESCRIPTION
**Pull request by cyc**
For some reason this option was added to `DNNLinearCombinedClassifier` and `DNNLinearCombinedRegressor` but not `DNNLinearCombinedEstimator`.

### CODE REVIEW

**Comment by karmel** [comment]
Thanks, @cyc , this seems reasonable to add. Can you add tests as well?

**Comment by cyc** [comment]
@karmel what did you have in mind? There are currently no existing tests for the `DNNLinearCombinedEstimator` class. The test that I have in mind would just invoke the constructor for `DNNLinearCombinedEstimator` and assert that the model fn is called with the proper value for the `batch_norm` argument, but that's kind of a weak test.

**Comment by karmel** [comment]
I believe we have some tests covering batch norm for the DNNs currently (eg https://github.com/tensorflow/estimator/blob/master/tensorflow_estimator/python/estimator/canned/dnn_testing_utils.py#L608 ) -- can we do something similar for the DNNLinearCombined?

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
